AdminCore.WebApi.Tests/Controllers/WorkingFromHomeControllerTests.cs

```diff
@@ -19,21 +20,26 @@ public class WorkingFromHomeControllerTests : BaseControllerTest
     private readonly IEventService _eventService;
     private readonly IFixture _fixture;
     private readonly IMapper _mapper;
+    private readonly EmployeeDto _employeeDto;
```

Comment:
This class is as far as I'm aware no longer used, we were waiting for the front end to confirm that they've switched over to using the EventController (previously WFH/Holidays/Sickness etc had their own controllers.)